platform/default/src/mbgl/storage/mbtiles_file_source.cpp

```diff
@@ -209,6 +211,7 @@ class MBTilesFileSource::Impl {
                 response.noContent = false;
                 response.expires = Timestamp::max();
                 response.etag = resource.url;
+                response.error.release();
```

Comment:
The memory pointed to by the unique_ptr is not cleaned up after `.release()`.

Reply:
I guess I don't know how to do that. Any tips?

Reply:
I think you can just `reset()` instead.

Reply:
Use `response.error.reset();` ?